version.ts

```diff
@@ -1,13 +1,12 @@
-import { resolve } from "https://deno.land/std@0.70.0/path/mod.ts"
 import { log } from "./src/utils.ts"
 
 const VERSION = "0.16.0"
 
+const { args } = Deno
```

Comment:
I'm not sure about the utility of this assignment. In my opinion this brings some noise to the code without really even improving readability or performance. I'd rather let it the way it is.